salt/base/motd.sls

```diff
@@ -0,0 +1,39 @@
+/etc/update-motd.d/10-help-text:
+  file.managed:
+    - mode: "0644"
+    - replace: True
```

Comment:
This results in empty files. Probably _ok_ but means we cannot easily revert this if we wanted to.